shared/src/types/vote_extensions/validator_set_update.rs

```diff
@@ -301,6 +301,38 @@ fn compute_hash(
     ])
 }
 
+/// Struct for serializing validator set
+/// arguments with ABI for Ethereum smart
+/// contracts.
+#[derive(Debug, Clone, Default)]
+pub struct ValidatorSetArgs {
+    /// Ethereum address of validators
+    pub validators: Vec<EthAddress>,
+    /// Voting powers of validators
+    pub powers: Vec<Uint>,
```

Comment:
Will the format of these voting powers be "normalized" uints? If so, we should document that, it could even have its own type in the namada codebase so we know where we are using it. 

I'm not sure if there is a proper term for that or what the proper term is. We normalize voting powers to be evenly distributed over some range (1 - 2**32), if I remember correctly

Reply:
No, these shouldn't normalized AFAIK

Reply:
ah actually, these should probably be normalized! I already have some code that does this here: https://github.com/anoma/namada/blob/b977845e4fdda64f571000c0508b72a1fca88ce0/shared/src/types/vote_extensions/validator_set_update.rs#L215-L224

Reply:
it could become a separate type, since it's being used in other places

Reply:
good to know! Will add a new type for this